lms/resources/_js_config/file_picker_config.py

```diff
@@ -31,14 +25,6 @@ def blackboard_config(cls, request, application_instance):
                 ),
             }
 
-        if groups_enabled:
-            config["listGroupSets"] = {
-                "authUrl": auth_url,
-                "path": request.route_path(
-                    "blackboard_api.courses.group_sets.list", course_id=course_id
-                ),
-            }
```

Comment:
Was this only used for file picker config or was this a mistake?

I'm wondering if something has been lost if this is file picker config to remove it from the file picker config file and put it unlabeled as such in the main one? Could we do with an extra `common_config()` or something?

Reply:
If this isn't picker stuff, that's great, because having two definitions of "enabled" here isn't very nice.

Reply:
I reckon this is not "filepicker" related in the sense used by the backend in this file.